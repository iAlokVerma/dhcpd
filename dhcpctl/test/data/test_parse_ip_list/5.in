344.2.2.1,256.123.0000001.42143479237432432497328974923847
