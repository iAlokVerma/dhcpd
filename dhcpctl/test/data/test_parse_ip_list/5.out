test_parse_ip_list: invalid input '344.2.2.1' from an IP address list
